src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -371,7 +364,7 @@ private async Task<IConfidentialClientApplication> BuildConfidentialClientApplic
                     authority = $"{ _applicationOptions.Instance}tfp/{_microsoftIdentityOptions.Domain}/{_microsoftIdentityOptions.DefaultUserFlow}";
                     app = ConfidentialClientApplicationBuilder
                         .CreateWithApplicationOptions(_applicationOptions)
-                        .WithRedirectUri(currentUri)
```

Comment:
nit: You can combine the 2 builders to avoid repeated code.

Reply:
good idea.

---
In reply to: [420316439](https://github.com/AzureAD/microsoft-identity-web/pull/144#discussion_r420316439) [](ancestors = 420316439)